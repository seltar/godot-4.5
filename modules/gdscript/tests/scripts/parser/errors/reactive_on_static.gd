extends RefCounted

# This should cause a parser error - @reactive on static variable
@reactive
static var invalid_static: int = 0

func test():
	print("This should not run due to parser error")
