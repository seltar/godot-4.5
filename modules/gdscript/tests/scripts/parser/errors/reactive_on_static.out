GDTEST_ANALYZER_ERROR
>> ERROR at line 4: "@reactive" annotation cannot be applied to a static variable.
